src/test/resources/partial-substations-map-data.json

```diff
@@ -1,26 +1,22 @@
 [
```

Comment:
It's not better to keep  { substations : [...] } like before ? I think it's a good practice, then we know what it contains.

Reply:
?? I didn't change anything here

Reply:
Ah OK, because before the front were receiving "{
  "substations": [ ... ]
}
but nvm not a big deal